packages/ckb-sdk-core/__mocks__/data/genesisBlock.json

```diff
@@ -0,0 +1,202 @@
+{
+  "header": {
+    "parentHash": "0x0000000000000000000000000000000000000000000000000000000000000000",
+    "transactionsRoot": "0x4d4d325247a64014ae5725dd562df408299687fc4dbd16e7ebb1d0a884b3b8ea",
+    "proposalsHash": "0x0000000000000000000000000000000000000000000000000000000000000000",
+    "witnessesRoot": "0xfd973862f00e91dc5d54a05ef167ea92ff30a7010f98bed36d321e47a86f6b9c",
+    "unclesHash": "0x0000000000000000000000000000000000000000000000000000000000000000",
+    "unclesCount": "0x0",
+    "chain_root": "0x0000000000000000000000000000000000000000000000000000000000000000",
```

Comment:
`chain_root` as been remove AFAIK.

Reply:
It will be updated by a real RPC call whenever CKB is released.

Reply:
We should still update 'mock' fixtures to reflect the fact that CKB `v0.21.x` doesn't have this field.

Reply:
The mocking data is updated in https://github.com/nervosnetwork/ckb-sdk-js/pull/356/commits/ce1f3a4c8eec02af1dfa18bc0eec0cd635d5eb69